flake.nix

```diff
@@ -6,7 +6,7 @@
     systems.url = "github:nix-systems/default";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "path:/Users/khuzema.khomosi/Documents/shared-kernel";
```

Comment:
See here for how to update shared-kernel:

https://github.com/nammayatri/nammayatri/blob/main/Backend/README.md#updating-flake-inputs

Reply:
Yea, Done the change... replaced the path to the remote repo and ran this command `nix flake lock --update-input shared-kernel --override-input shared-kernel github:nammayatri/shared-kernel/f718aabf4ca993a26063b64e3712ee1ad2451bf1`